### ISSUE TITLE
[Bug]: Not enough disk space error shown in Genshin Impact Installer

### ISSUE THREAD

**Issue by dyzdyz010** [Whisky-App/Whisky#443]
### Description

<img width="1300" alt="image" src="https://github.com/Whisky-App/Whisky/assets/1324420/f49b20f6-6040-40e9-9ad8-762fad1676c5">

Translation: Not enough space in selected destination.

### Steps to reproduce

1. Install Genshin Impact Launcher
2. Open launcher and install game
3. Error pops up

### Expected behaviour

Install the game without errors

### Logs

```bash
Whisky Version: 2.0.0
Date: 2023/9/27 15:21:16
macOS Version: 版本14.0（版号23A344）
Bottle Name: Bottle
Wine Version: 7.7.0
Windows Version: win10
Bottle URL: /Users/dyz/Library/Containers/com.isaacmarovitz.Whisky/Bottles/1ADC72D3-B394-49B5-B996-D398CE3A251F

Arguments: start /unix /Users/dyz/Library/Containers/com.isaacmarovitz.Whisky/Bottles/1ADC72D3-B394-49B5-B996-D398CE3A251F/drive_c/Program Files/Genshin Impact/launcher.exe 

Launched Wine (98731)
```

### What version of Whisky are you using?

2.0.0

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by boa-z** [comment]
just not enough space... need 146g but only 144g available

**Comment by wangxy98c** [comment]
Yes, I have encountered the same issue. The computer has over 200GB of available space, but it seems like the container only allows the use of 66GB of space, and I'm unable to install."Have you resolved the issue?

**Comment by boa-z** [comment]
I didn’t “solve” it. In fact, I downloaded the game and copy the game data to my bottle. In that way, no error occurred.

**Comment by xuhao1** [comment]
I have same issue here. It's seems to be iCloud issue: It count the icloud cloud online file into your used disk.

**Comment by greatoe** [comment]
> I didn’t “solve” it. In fact, I downloaded the game and copy the game data to my bottle. In that way, no error occurred.

Would you please advise how to download all the game files?  I can only download the installer, which is only 100+Mb in size.
